.github/workflows/rust.yml

```diff
@@ -46,6 +46,21 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           args: --all-features --all-targets -- -Dwarnings -Drust-2018-idioms
 
+  clippy-tools-output:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          components: clippy
+          override: true
+      - uses: actions-rs/clippy-check@v1
+        with:
+          name: clippy-tools-results
+          token: ${{ secrets.GITHUB_TOKEN }}
+          args: --manifest-path tools/kvstore-test/Cargo.toml --all-features --all-targets -- -Dwarnings -Drust-2018-idioms
```

Comment:
I wonder if this should also be made into a Cargo alias so it's easy to run locally too.

Reply:
Yeah had the same intuition. There is a follow-up PR with a lot of improvements to the consistency of those commands.